packages/subgraph/src/mappingHelpers.ts

```diff
@@ -338,6 +340,16 @@ export function getOrInitFlowOperator(
         flowOperatorEntity.flowOperator = flowOperatorAddress;
         flowOperatorEntity.updatedAtBlockNumber = block.number;
         flowOperatorEntity.updatedAtTimestamp = currentTimestamp;
+
+        // It will only be called when the entry does not exist. Previously, it was being called every time the user updated their permission or flow allowance
+        const superTokenContract = ISuperToken.bind(
+            tokenAddress
+        );
+        const currentAllowance = superTokenContract.try_allowance(senderAddress, flowOperatorAddress);
```

Comment:
This is for the first event when the user doesn't exist.  What if someone gives allowance before ACL permission then because of handleApprove implementation if ignore `if (!flowOperator) return` and maybe we will see the `allowance` discrepancy.